### ISSUE TITLE
Swipe selection

### ISSUE THREAD

**Issue by gliviu** [AnySoftKeyboard/AnySoftKeyboard#509]
Add option to move cursor when swiping the keyboard.
Similar to https://www.youtube.com/watch?v=6h2yrBK7MAY

**Comment by timofonic** [comment]
This would be very interesting, I agree...

**Comment by homersimpsons** [comment]
It already exist in gesture you can choose for swap left and right the option move cursor left/right
If it is what you were looking for this ticket could be close.

**Comment by gliviu** [comment]
I know about that feature. It is not as convenient as the one initially suggested.

**Comment by homersimpsons** [comment]
Yes, that is not exactly the same ... but I don't know if can be implement on smartphone because of the sensitivity

**Comment by gliviu** [comment]
Samsung has this feature on it's own phones as well using the proprietary samsung keyboard. But I agree this might not work well for smaller screens. If this is too difficult to implement I agree it should be closed.

**Comment by homersimpsons** [comment]
I think it's better to keep and maybe set screen dimension limit to be able to use this or make a warning for small screen

**Comment by modx-arseniy** [comment]
https://play.google.com/store/apps/details?id=uk.me.fantastic.anysoftkeyboard&hl=ru&referrer=utm_source%3Dgoogle%26utm_medium%3Dorganic%26utm_term%3Danysoftkeyboard+v111++swipeselection+github&pcampaignid=APPU_1_my-uXMnsCPGemwWZzr_YCw

we have it, yeah! but no localisation. -(

**Comment by PrSunflower** [comment]
> 
> 
> https://play.google.com/store/apps/details?id=uk.me.fantastic.anysoftkeyboard&hl=ru&referrer=utm_source%3Dgoogle%26utm_medium%3Dorganic%26utm_term%3Danysoftkeyboard+v111++swipeselection+github&pcampaignid=APPU_1_my-uXMnsCPGemwWZzr_YCw
> 
> we have it, yeah! but no localisation. -(

@modx-arseniy Cool! Is it a keyboard or an add-on that can work with AnySoftKeyboard?

**Comment by menny** [comment]
It's a fork. So, a different app you need to install and use.

And I see that it's a fork from 2013.
